src/machines/java/com/enderio/machines/common/blockentity/solar/SolarPanelBlockEntity.java

```diff
@@ -1,5 +1,6 @@
 package com.enderio.machines.common.blockentity.solar;
 
+import com.enderio.EnderIO;
```

Comment:
remove import

Reply:
oooouh forgot to remove it when debugging with the logger...